src/ledger/LedgerManagerImpl.cpp

```diff
@@ -666,8 +680,15 @@ LedgerManagerImpl::closeLedger(LedgerCloseData const& ledgerData)
     {
         releaseAssert(ledgerCloseMeta);
         ledgerCloseMeta->v0().ledgerHeader = mLastClosedLedger;
-        mMetaStream->writeOne(*ledgerCloseMeta);
-        mMetaStream->flush();
+        if (!mApp.getConfig().UNSAFE_EMIT_META)
```

Comment:
as you're handling both conditions, there is no reason to use the negative here?

Reply:
True.  Depending on which interfaces from among the current (or future) suggestions we pick, that path may or may not end up with two branches; if it does I'll put the positive branch first.

Reply:
(In light of other changes, there's now only one branch anyway.)